twisted/topfiles/8731.bugfix

```diff
@@ -0,0 +1 @@
+Add helper to create bytes reprs that always begin with b.  twistd on UNIX platforms can report non-ASCII errors on Python 2 and all errors on Python 3 encountered during daemonization.
```

Comment:
`Add helper to create bytes reprs that always begin with b.` ...  which helper? 
Please note that the release notes are provided out of the context of this commit .
Also, this is about a private API and the release notes should not contain private stuff